tide/master/rest/RestInterface.cpp

```diff
@@ -69,9 +70,9 @@ namespace
 struct HtmlInterface
 {
     HtmlInterface(http::Server& server, DisplayGroup& group,
-                  const MasterConfiguration& config)
+                  const MasterConfiguration& config, bool& locked)
```

Comment:
no & for bool here! Just const bool, you're not going to modify the bool passed to the constructor